### PR TITLE
Add marker support in logging filters #26544

### PR DESCRIPTION
**Pull request by gbrd**

### CODE REVIEW

**Comment by raboof** [comment]
OK TO TEST

**Comment by johanandren** [comment]
Refs #26544

File: akka-actor/src/main/scala/akka/event/Logging.scala [hunk 1]
```diff
@@ -1466,13 +1472,18 @@ class MarkerLoggingAdapter(
   def this(bus: LoggingBus, logSource: String, logClass: Class[_]) =
     this(bus, logSource, logClass, new DefaultLoggingFilter(() ⇒ bus.logLevel))
 
+  def isErrorEnabled(marker: LogMarker) = loggingFilter.isErrorEnabled(logClass, logSource, marker)
+  def isWarningEnabled(marker: LogMarker) = loggingFilter.isWarningEnabled(logClass, logSource, marker)
+  def isInfoEnabled(marker: LogMarker) = loggingFilter.isInfoEnabled(logClass, logSource, marker)
+  def isDebugEnabled(marker: LogMarker) = loggingFilter.isDebugEnabled(logClass, logSource, marker)
+
```
Comment (patriknw) [inline akka-actor/src/main/scala/akka/event/Logging.scala]: add similar in https://github.com/akka/akka/blob/master/akka-actor-typed/src/main/scala/akka/actor/typed/Logger.scala#L170
and https://github.com/akka/akka/blob/master/akka-actor-typed/src/main/scala/akka/actor/typed/internal/adapter/LoggerAdapterImpl.scala#L366

**Review by patriknw — COMMENTED** [review]
didn't know about that but seems like a good addition

File: akka-actor/src/main/scala/akka/event/Logging.scala (hunk 1 above)
Comment (gbrd) [reply akka-actor/src/main/scala/akka/event/Logging.scala]: done

**Comment by gbrd** [comment]
Should we add a separate trait (and config key) to solve the binary compatibility problem ?

If yes should it contain only methods with marker or all methods in order to replace/depreciate the other trait completely ?

**Review by patriknw — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]
LGTM, thanks @gbrd

**Pull request merged**